content/tracing/setup/php.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tracing PHP Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-php"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+---
+
+PHP APM is currently in **alpha**** and not officially supported by Datadog. The officially supported PHP Beta will launch as early as September 2018.  To be notified of the beta launch, [submit this form][1].
```

Comment:
there are two extra asterisks in `**alpha****`

Reply:
How about the following (so it mirrors the .NET page):

"The APM tracer for PHP applications is currently in **alpha** and not officially supported by Datadog. The officially supported beta will launch as early as September 2018. To be notified of the beta launch, [submit this form][1]."

Reply:
Also can we make this section an alert like we do in https://docs.datadoghq.com/tracing/setup/javascript/?